renderdoc/core/core.cpp

```diff
@@ -319,7 +319,7 @@ RenderDoc::~RenderDoc()
   if(m_RemoteThread)
   {
     m_RemoteServerThreadShutdown = true;
-    // don't join, just close the thread, as we can't wait while in the middle of module unloading
```

Comment:
I dont quite understand why we cant join the thread whilst the module is unloading. It seems like the proper thing to do, and was required for me for the RenderDoc layer to unload & load cleanly on Android.